managed/services/backup/removal_service_test.go

```diff
@@ -43,8 +45,7 @@ func TestDeleteArtifact(t *testing.T) {
 	removalService := NewRemovalService(db, mockedPbmPITRService)
 
 	agent := setup(t, db.Querier, models.MySQLServiceType, "test-service")
-	endpoint := "https://s3.us-west-2.amazonaws.com/"
-	accessKey, secretKey, bucketName, bucketRegion := "access_key", "secret_key", "example_bucket", "us-east-2"
+	accessKey, secretKey, bucketName, bucketRegion := "access_key", "secret_key", "example_bucket", "us-east-2" //nolint:goconst
```

Comment:
it seems like these variables are also can be moved to the top as constants

Reply:
I did that, but in another PR :) So I'll revert here.

Reply:
https://github.com/percona/pmm/pull/2215/files#diff-44238c72dea7ccc289d4c97c9b2334927d0d6349232641af9188afdf1f11fabaR46-R52